### DISCUSSION TITLE
Provide a fly.io scale to zero example

### DISCUSSION THREAD

**Discussion by KyleAMathews** [electric-sql/electric#843] [Feature requests]
Fly makes it pretty easy to build low cost on demand services. Ideal for many electric apps which wouldn't have constant usage as the electric service only needs to be running when there's an active user.

To support this, the electric docker instance would need to shut down after some period of no connections e.g. https://github.com/fly-apps/autoscale-to-zero-demo

**Comment by thruflo** [comment]
This is a nice idea -- and would tie into an upgraded Fly guide that factors in the Supabase <> Fly Postgres development. Perhaps we could do a scale to zero demo and tie into an updated deployment recipe, cc @msfstef @alco.

**Reply by msfstef** [reply]
Interesting - and likely to be popular for smaller deployments. Would we add it as an option/banner in the existing deployment recipe?

**Reply by thruflo** [reply]
Yes but I think we need to iterate the Fly recipe given the changes with their Postgres offering. Ideally Digital Ocean, Fly+Supabase and AWS could all have upgraded guides, as per the deployment page mock up.

**Comment by KyleAMathews** [comment]
👋 we've been working the last month on a rebuild of the Electric server over at a temporary repo https://github.com/electric-sql/electric-next/

You can read more about why we made the decision at https://next.electric-sql.com/about

We're really excited about all the new possibilities the new server brings and we hope you'll check it out soon and give us your feedback.

We're now moving the temporary repo back here. As part of that migration we're closing all the old issues, PRs, and discussions. We really appreciate you taking the time to start this discussion! If it's still relevant in the new system, please reopen.
